pkg/config/types.go

```diff
@@ -32,7 +32,7 @@ type Cloudfront struct {
 	SecureProtocol bool
 }
 
-type source struct {
+type CloudSource struct {
```

Comment:
How about only `Source`? We've an example in docs which reads from local filesystem, that doesn't really fit in the `CloudSource` definition.

Reply:
can't use Source because there is a function called Source() to get the private source from config.
Named it CloudSource because saw there is 3 usage of this source which is `WebFolder`, `S3`, and `CloudFront` Do you have better naming in mind? 

How about `DataSource`?

Reply:
`DataSource` should be good.

Reply:
Renamed it to `DataSource` in the config struct itself, so we don't need to change the configuration file convention. Please check, and verify